src/org/opendatakit/briefcase/ui/reused/source/ShowSourceForm.java

```diff
@@ -32,26 +32,44 @@
   private JLabel sourceLabel;
   private JButton resetButton;
   private JLabel actionLabel;
+  private JButton reloadButton;
   private List<Runnable> onResetCallbacks = new ArrayList<>();
+  private List<Runnable> onReloadCallbacks = new ArrayList<>();
+  private Operation type;
 
-  private ShowSourceForm(String action) {
+  private ShowSourceForm(String action, Operation type) {
```

Comment:
I personally **love** enums but maybe, in this case, maybe a `boolean showReloadButton` could suffice. What do you think? Same value for less abstractions :)

Reply:
Sure, the boolean's name does immediately make the parameters intention clearer than the use of an enum.